site/content/api/Event-API.md

```diff
@@ -76,4 +81,63 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+In the following these events (currently one) are specified in detail.
+
+
+### Device command readiness notification
+
+By sending an event of this type the receiver is notified that a device is ready to receive a command.
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
```

Comment:
IMHO we should remove this sentence since it assumes a particular way of implementation which may or may not be realized this way. In any case, though, the implementation is not relevant for describing the purpose of this event.

Reply:
Agreed - is contained also in the concept page anyway.